BaseClasses.py

```diff
@@ -1338,7 +1338,7 @@ def add_exits(self, exits: Union[Iterable[str], Dict[str, Optional[str]]],
 
         :param exits: exits from the region. format is {"connecting_region": "exit_name"}. if a non dict is provided,
         created entrances will be named "self.name -> connecting_region"
```

Comment:
formatting noticed when testing sphinx docs, without this spacing it doesn't recognize the following param tag
```suggestion
        :param exits: exits from the region. format is {"connecting_region": "exit_name"}. if a non dict is provided,
                      created entrances will be named "self.name -> connecting_region"
```